apps/src/studio/levels.js

```diff
@@ -2551,19 +2553,22 @@ levels.js_hoc2015_add_characters = {
     'addCharacter("PufferPig");',
     '',
     'function whenGetPufferPig() {',
-    '  playSound("PufferPigSound1");',
+    '  playSound("PufferPigRandom");',
```

Comment:
Just matching the original capitalization change, where we didn't capitalize sound in `R2-D2sound1` because it followed a number, but we did in `PufferPigSound1` because it did not. All of the values are case-insensitive in the lower level API, but these do need to match case with the `skin.soundGroups` entry for `randomValue`